Simplenote/MagicLinkConfirmationView.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+import SwiftUI
+
+
+// MARK: - MagicLinkConfirmationView
+//
+struct MagicLinkConfirmationView: View {
+    
+    @State var displaysInvalidLink = false
+    
+    var onDismissRequest: (() -> Void)?
+    
+    var body: some View {
+        VStack(spacing: 20) {
```

Comment:
Nitpick: You got a lot of inline constants in this file

Reply:
Addressed, TY!